testtools/compat.py

```diff
@@ -86,6 +85,7 @@ def classtypes():
         return (type, types.ClassType)
     str_is_unicode = sys.platform == "cli"
     text_or_bytes = (unicode, str)
+    text = unicode
```

Comment:
We don't actually, at least, not a direct one. Want me to _add_ a dependency on six as part of this PR?